### ISSUE TITLE
Alpha channel for color behaves inconsistently between web and desktop

### ISSUE THREAD

**Issue by RybekU** [ryanisaacg/quicksilver#606]
**Description**
Using Color::with_alpha on web and locally produces different results as illustrated on image below:
![obraz](https://user-images.githubusercontent.com/45723292/81461184-436e6e80-91aa-11ea-90a3-d5cfe3fa2fc9.png)
Left rectangle  - web
Right rectangle - desktop application
**To Reproduce**
Background color is `Color::CYAN`.
Rectangle is created using:
```rust
  gfx.fill_rect(&area, Color::BLUE.with_alpha(0.2));
  gfx.stroke_rect(&area, Color::BLUE);
```

**Environment and versions:**
- Environment:  Windows 10 and Firefox
- Rust compiler version:  stable
- Quicksilver version:  commit rev = "8fdff10"

**Comment by ryanisaacg** [comment]
That's... interesting! I'll read the specs on blending for WebGL vs Desktop GL to see if there's anything that jumps out at me. I hope this is an easy fix but I can't promise that it will be.

**Comment by lenscas** [comment]
So... here is an interesting one. I decided to check the difference for myself mostly because I was curious and although the build for native looks quite similar to what is on the right. When I use webstd to build for the web the square seems to become green

![image](https://user-images.githubusercontent.com/8694039/82104363-ef6a0980-9716-11ea-88b3-da7b60cc2137.png)

Also, if I change the background color to white, the square becomes yellow instead.

![image](https://user-images.githubusercontent.com/8694039/82104524-96e73c00-9717-11ea-9cb5-767b1ef1a9b5.png)

Browser:76.0.1 (64-bit)
OS: Linux
Quicskilver version: d1057573afc0d78104406dc956c09fe02f28ff22

In order to produce this, I simply cloned quicksilver and changed the 01_example to contain the above mentioned code.

**Comment by AnthonyYoManz** [comment]
Pretty sure this is caused by the alpha-blending happening against the web-page. If you change the <body>'s background color to white and run some code that draws a semi-transparent black square over a sprite then the sprite will be blended with white. If you change the <body>'s background colour to black and repeat the process, the sprite will be blended with black.

**Comment by ryanisaacg** [comment]
Both issues here should be resolved by ryanisaacg/blinds#28: the alpha should show up the same on desktop and web, and the canvas should not be blended with the page background.

**Comment by ryanisaacg** [comment]
Let me know if either of these issues crop up after you upgrade dependencies, but the new blinds version should take care of them.
